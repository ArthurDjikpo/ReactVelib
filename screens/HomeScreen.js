import React from 'react';
import { StyleSheet, ImageBackground, Button } from 'react-native';


export default function HomeScreen({navigation}) {
  return (
    <>
      <ImageBackground style={ styles.container } 
        resizeMode='cover' 
        source={{uri: 'https://voiturelectrique.eu/wp-content/uploads/2019/09/voiturelectrique.eu_.Jump-by-Uber2019-09-01-17.11.05.jpg'}}/>
        <Button
          title="Go to Map"
          onPress={() => navigation.navigate('Links')}
        />
        <Button
          title="Go to List"
          onPress={() => navigation.navigate('Velib')}
        />
    </>

  )}

HomeScreen.navigationOptions = {
  title: 'Bienvenue ! '
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    width: null,
    height: null,
  },
});
