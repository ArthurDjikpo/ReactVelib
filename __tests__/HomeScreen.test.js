import React from 'react';
import renderer from 'react-test-renderer';
import HomeScreen from '../screens/HomeScreen'

test('Home snapShot', ()=> {
    const snap = renderer.create(
        <HomeScreen />
    );
    let tree = snap.toJSON();
    expect(tree).toMatchSnapshot();
})