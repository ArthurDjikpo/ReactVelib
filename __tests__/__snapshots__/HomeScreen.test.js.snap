// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapShot 1`] = `
Array [
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "flex": 1,
        "height": null,
        "width": null,
      }
    }
  >
    <Image
      resizeMode="cover"
      source={
        Object {
          "uri": "https://voiturelectrique.eu/wp-content/uploads/2019/09/voiturelectrique.eu_.Jump-by-Uber2019-09-01-17.11.05.jpg",
        }
      }
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": null,
            "width": null,
          },
          undefined,
        ]
      }
    />
  </View>,
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Go to Map
      </Text>
    </View>
  </View>,
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Go to List
      </Text>
    </View>
  </View>,
]
`;
